test/e2e/pkg/testnet.go

```diff
@@ -482,6 +485,39 @@ func (t Testnet) HasPerturbations() bool {
 	return false
 }
 
+func (t Testnet) prometheusConfigBytes() ([]byte, error) {
+	tmpl, err := template.New("prometheus-yaml").Parse(`global:
+  scrape_interval: 1s
+
+scrape_configs:
+{{- range .Nodes }}
+  - job_name: '{{ .Name }}'
+    static_configs:
+      - targets: ['localhost:{{ .PrometheusProxyPort }}']
+{{end}}`)
```

Comment:
The template would be better put in a standalone file.

Reply:
Done. Thanks for the suggestion!